.github/workflows/checklist.yml

```
@@ -0,0 +1,16 @@
+name: Add comment
+
+on:
+- pull_request_target
+
+jobs:
+  comment:
+    name: Checklist
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v0.1.0
+    - uses: harupy/comment-on-pr@v0.1.0
```

We should be even more strict, and pin a commit not the tag.

A tag can be updated with a simple write, so you end up with different souce code for the same tag.

The source code for the same commit sha can only be changed if you rewrite the whole history, and find a way to generate the exact same  sha.

```suggestion
    - uses: harupy/comment-on-pr@d83982f39ec865c951b0d1ad0e9e0d79d801e790
```